pulsar-common/src/main/proto/PulsarApi.proto

```diff
@@ -193,6 +193,9 @@ message CommandSubscribe {
     repeated KeyValue metadata = 10;
 
     optional bool read_compacted = 11;
+	// Signal wthether the subscription will initialize on latest
+	// or not -- earliest
+	optional bool initialize_on_latest = 12 [default = true];
```

Comment:
There is already `start_message_id` field, which is currently only used by the `Reader` implementation. We should use that (passing either `MessageId.earliest` or `MessageId.latest`) when subscribing.

Reply:
As far as I know about pulsar, the nonpersistent topic subscription (NonpersistentTopicSubscription) use the `start_message_id` field when subscribing. And the persistent topic has no business with it. Therefore, I didn't use that. Do I misunderstand it?

PS. I hope that I don't use nonproper expressions and offend you since I'm not a native English speaker.